packages/socat/0001-xioopts-conditionally-compile-applyopts_termios_valu.patch

```diff
@@ -0,0 +1,59 @@
+From 2af3558817b3ee55b2aa8483aabd634719456ad7 Mon Sep 17 00:00:00 2001
+From: Kush Upadhyay <kushupad@amazon.com>
+Date: Wed, 12 Jun 2024 21:40:54 +0000
+Subject: [PATCH] xioopts: conditionally compile applyopts_termios_value
+ function
+
+Signed-off-by: Kush Upadhyay <kushupad@amazon.com>
+---
+ xioopts.c | 34 ++++++++++++++++++----------------
+ 1 file changed, 18 insertions(+), 16 deletions(-)
+
+diff --git a/xioopts.c b/xioopts.c
+index 4b651aa..2aa2bed 100644
+--- a/xioopts.c
++++ b/xioopts.c
+@@ -4041,22 +4041,24 @@ int applyopt_spec(
+ 	return 0;
+ }
+ 
+-int applyopts_termios_value(
+-	int fd,
+-	struct opt *opt)
+-{
+-	 if (((opt->value.u_uint << opt->desc->arg3) & opt->desc->minor) !=
+-	     (opt->value.u_uint << opt->desc->arg3)) {
+-	    Error2("option %s: invalid value %u",
+-		   opt->desc->defname, opt->value.u_uint);
+-	    return -1;
+-	 }
+-	 if (xiotermios_value(fd, opt->desc->major, opt->desc->minor,
+-			      (opt->value.u_uint << opt->desc->arg3) & opt->desc->minor) < 0) {
+-	    return -1;
+-	 }
+-	return 0;
+-}
++#if WITH_TERMIOS
++	int applyopts_termios_value(
```

Comment:
Convention is not to indent the lines guarded by `#ifdef` - that keeps the code readable in the case where it's used, and for the purposes of this patch will minimize the delta.

Reply:
Got it, will remove those indents